css/input.css

```diff
@@ -0,0 +1,124 @@
+
+
+.login_bar {
+	width: 300px;
+	height:45px;
+	background-color:#6B90D9;
+	border-radius: 10px;
+	position: relative;
+    z-index: 1;
+	overflow: hidden;
+	
+	
+}
+
+.login {
+	background-color:#6B90D9;
+	color:#fff;
+	position: absolute;
+	height: 103%;

```

Comment:
set the **height: 100%,** the problem with the white stuff is not going to be solve when setting the height to 103%. the white thing is the default border of an input , set the **border: none** and there you go

Reply:
thanks! done.